app/models/miq_server/worker_management/monitor/settings.rb

```diff
@@ -51,4 +61,23 @@ def get_time_threshold(worker)
   def get_memory_threshold(worker)
     @child_worker_settings[worker.class.settings_name][:memory_threshold]
   end
+
+  def get_additional_settings(worker_settings_paths)
+    additional_settings = {}
+    worker_settings_paths.to_a.each do |settings_path|
+      additional_settings.store_path(settings_path, Settings.to_hash.dig(*settings_path))
+    end
+
+    additional_settings
```

Comment:
sorry, distraction:

```suggestion
    worker_settings_paths.to_a.each_with_object({}) do |settings_path, additional_settings|
      additional_settings.store_path(settings_path, Settings.to_hash.dig(*settings_path))
    end
```

Is this basically just producing a subset of the settings (picking out worker_settings keys)?

maybe the title "additional_setting" is confusing for me